__tests__/app.test.js

```diff
@@ -74,6 +74,115 @@ describe(`NCNews-Server Unit Tests`, () => {
         });
 
            
+    });
+
+    // Error Handling
+    describe("Error Handling Tests", () => {
+        // GET METHOD 404 ERROR TEST
+        test("[ 404 ] Responds with a 404 error when an invalid path is specified", () => {
+          return request(app).get("/sdfhdshifsdhfsd").expect(404).then((response) => {
+              expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+            });
+        });
+        // need 500 internal server error test - find out what to do for that one
+    });
+
+    // Task 5 - get api articles articleid
+
+    describe(`GET /api/articles/:article_id`, () => {
+
+        test(`[ 200 ] Responds with an object with key article and keyval of an object with the following 7 properties: author, title, article_id, topic, body, created_at, votes`, () => {
+
+            return request(app).get(`/api/articles/1`).expect(200).then((response) => {
+                const article = response.body.article;
+
+                    expect(article).toEqual(
+                        expect.objectContaining({
+                            author: expect.any(String),
+                            title: expect.any(String),
+                            topic: expect.any(String),
+                            body: expect.any(String),
+                            article_id: expect.any(Number),
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                    }));
+            });
+        });
+
+        test(`[ 200 ] Responds with the correct article object with correct articleID`, () => {
+
+            return request(app).get(`/api/articles/2`).expect(200).then((response) => {
+                const article = response.body.article;
+                expect(article[`article_id`]).toBe(2);
+            });
+
+        });
+
+        test(`[ 404 ] Responds with an error when passed invalid parameters`, () => {
+
+            return request(app).get(`/api/articles/sdfdefds`).expect(404).then((response) => {
+                expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+            });
+
+        });
+
+        test(`[ 404 ] Responds with an error when passed an SQL injection test 1`, () => {
+
+            return request(app).get(`/api/articles/1&#59;&nbsp;DROP&nbsp;TABLE&nbsp;articles`).expect(404).then((response) => {
+                expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+            });
+
+        });
+
+        test(`[ 404 ] Responds with an error when passed an SQL injection test 2`, () => {
+
+            return request(app).get(`/api/articles/1; DROP TABLE articles`).expect(404).then((response) => {
+                expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+            });
+
+        });
+
+
+    });
+
```

Comment:
You haven't checked for a 404 as the above tests are all bad requests. Add a test for a valid but non-existent article id.